src/ui/Quickview/QuickviewDocumentIframe.ts

```diff
@@ -32,7 +32,7 @@ export class QuickviewDocumentIframe {
     return meta && meta.getAttribute('content') == 'pdf2htmlEX';
   }
 
-  public render(htmlDocument: HTMLDocument): Promise<HTMLIFrameElement> {
+  public render(htmlDocument: HTMLDocument, title?: string): Promise<HTMLIFrameElement> {
```

Comment:
Hmm, I think the `QuickviewDocumentIframe` class is not publicly available on the Coveo namespace. If so, we could make the argument required.

Reply:
Alright, I'll make it required.